lib/blueshift/test.js

```diff
@@ -94,57 +94,77 @@ describe('Blueshift', function(){
         analytics.stub(window.blueshift, 'identify');
       });
 
-      it('should send an id', function(){
+      it('should call identify with an id', function(){
         analytics.identify('id');
-        analytics.called(window.blueshift.identify, { id: 'id', _bsft_source: 'segment.com' });
+        analytics.called(window.blueshift.identify, {
+          id: 'id',
+          customer_id: 'id',
+          _bsft_source: 'segment.com',
+          anonymousId: analytics.user().anonymousId()
+        });
       });
 
-      it('should not call identify if an id is not sent', function(){
+      it('should call identify even if id is missing', function(){
         analytics.identify();
-        analytics.didNotCall(window.blueshift.identify);
-      });
-
-      it('should not send only traits', function(){
-        analytics.identify({ trait: true });
-        analytics.didNotCall(window.blueshift.identify);
+        analytics.called(window.blueshift.identify, {
+          _bsft_source: 'segment.com',
+          anonymousId: analytics.user().anonymousId()
+        });
       });
 
-      it('should send an id and traits', function(){
-        analytics.identify('id', { trait: true });
-        analytics.called(window.blueshift.identify, { id: 'id', trait: true, _bsft_source: 'segment.com' });
-      });
     });
 
-    describe('#group', function(){
+    describe('#track', function(){
       beforeEach(function(){
         analytics.stub(window.blueshift, 'track');
       });
 
-      it('should call track with event equal group', function(){
-        analytics.group('123', { name: 'test',  industry: 'Technology' });
-        analytics.called(window.blueshift.track, 'group', {
-          name: 'test',
-          industry: 'Technology',
-          id: '123',
-          _bsft_source: 'segment.com'
-        });
+      it('should call track event with anonymousId and/or user id', function(){
+        analytics.track('event');
+        var properties = {
+          _bsft_source: "segment.com",
+          anonymousId: analytics.user().anonymousId(),
+          customer_id: analytics._user.id() || undefined,
+        }
+        removeBlanks(properties);
```

Comment:
Rather than doing dynamic filtering in tests, be explicit about what we're testing here